test/replay/covenant_sender.go

```diff
@@ -163,18 +166,21 @@ func (c *CovenantSender) SendCovenantSignatures() {
 	}
 }
 
-func signStakeExpansionTx(t *testing.T, covenantSK *btcec.PrivateKey, del *bstypes.BTCDelegationResponse, infos *StakingInfos) *types.BIP340Signature {
+func signStakeExpansionTx(t *testing.T, covenantSK *btcec.PrivateKey, del *bstypes.BTCDelegationResponse, infos, prevDelInfos *StakingInfos) *types.BIP340Signature {
 	fundingTxBz, err := hex.DecodeString(del.StkExp.OtherFundingTxOutHex)
 	require.NoError(t, err)
 	otherFundingTxOut, err := btcstaking.DeserializeTxOut(fundingTxBz)
 	require.NoError(t, err)
 
+	prevDelUnbondPathSpendInfo, err := prevDelInfos.StakingSlashingInfo.StakingInfo.UnbondingPathSpendInfo()
```

Comment:
should this be `StakingSlashingInfo` ?

Reply:
it is actually the stakingInfo, for some reason in the test utils, it is within the[ StakingSlashingInfo](https://github.com/babylonlabs-io/babylon/blob/57fde9216a81999f3a3c16a1b0459083ec391f70/testutil/datagen/btcstaking.go#L476-L477)